presentation-api/controlling-ua/PresentationAvailability_onchange.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Monitoring the list of available presentation displays.</title>
+<link rel="author" title="Tomoyuki Shimizu" href="https://github.com/tomoyukilabs">
+<link rel="help" href="http://w3c.github.io/presentation-api/#monitoring-the-list-of-available-presentation-displays">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="common.js"></script>
+
+<p id="notice">The test passes if a "PASS" result appears.<br></p>
+
+<script>
+    // prevent the default timeout
+    setup({explicit_timeout: true});
+
+    var notice = document.getElementById('notice');
+
+    promise_test(function(t) {
+      // clean up the instruction notice when the test ends
+      t.add_cleanup(function() {
+          notice.style.display = 'none';
+      });
+
+      // initialize a presentation request
+      var request = new PresentationRequest(presentationUrls);
+
+      var availability, previousState, eventWatcher;
+
+      // set timeout to observe the presentation availability
+      t.step_timeout(function() {
+        t.force_timeout();
+        t.done();
+      }, 90000);
+
+      var wait = t.step_func(function(a) {
+        availability = a;
+
+        // save the current value of the presentation availability
+        previousState = availability.value;
+
+        // add an instruction notice
+        notice.innerHTML += 'Please make your presentation display '
```

Comment:
The user may have more than one presentation display to unplug, I would use a plural form in "Please make your presentation displays..."